test/box-luatest/lua_memory_test.lua

```diff
@@ -0,0 +1,68 @@
+local server = require('luatest.server')
+local t = require('luatest')
+local g = t.group('lua_memory setting test')
+--local treegen = require('test.treegen')
+
+local MB = 1024 * 1024
+local LUA_MIN_MEMORY = 256 * MB
+
+g.before_each(function()
+    g.server = server:new{
+        alias = 'default',
+        box_cfg = {lua_memory = LUA_MIN_MEMORY}
+    }
+    g.server:start()
+
+end)
+
+g.after_each(function()
+    g.server:drop()
+end)
+
+g.test_lua_memory_setting = function(g)
+    g.server:exec(function()
+        local MB = 1024 * 1024
+        local LUA_MIN_MEMORY = 256 * MB
+        local LUA_EXTENDED_MEMORY = 600 * MB
+
+        t.assert_equals(box.cfg.lua_memory, LUA_MIN_MEMORY)
+
+        -- Increasing the limit should be ok.
+        box.cfg{lua_memory = LUA_MIN_MEMORY + 1}
+        t.assert_equals(box.cfg.lua_memory, LUA_MIN_MEMORY + 1)
+
+        local err_min = 'Cannot limit the Lua memory with values less than '..
```

Comment:
It is better to surround operators (`..` in that case) with spaces [1].
Here and below.
[1]: https://www.tarantool.io/en/doc/latest/contributing/lua_style_guide/

Reply:
Done.